src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -1984,11 +1733,16 @@ internal Type ResolveTypeName(string typeName)
                 var baseType = ResolveTypeName(baseName + typeArgs.Count);
                 return baseType.MakeGenericType(typeArgs.ToArray<Type>());
             }
-
+            
             throw new TypeAccessException("Type string \"" + typeName + "\" cannot be resolved.");
         }
 
 #endregion
+        [AttributeUsage(AttributeTargets.ReturnValue)]
+        public class StreamedAttribute : Attribute
+        {
+        }
+        [return: Streamed]
```

Comment:
what is this for?

Reply:
oops I was just playing with attributes 😱